packages/suite/src/hooks/wallet/coinmarket/form/common/useCoinmarketFormActions.ts

```diff
@@ -64,7 +65,7 @@ export const useCoinmarketFormActions = <T extends CoinmarketSellExchangeFormPro
 }: CoinmarketUseFormActionsProps<T>): CoinmarketUseFormActionsReturnProps => {
     const { symbol } = account;
     const { shouldSendInSats } = useBitcoinAmountUnit(symbol);
-    const accounts = useSelector(selectAccounts);
+    const { accounts } = useAccounts();
```

Comment:
what is the reason for this refactor? I do not see benefit of using this hook

Reply:
oh I see. Please revert it

Reply:
Since we agreed to not filter already discovered accounts I will remove this change